src/main/java/org/jabref/gui/integrity/BibLogSettingsPane.fxml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import java.lang.*?>
```

Comment:
Is it possible to have non-star imports, here, too?

I think with our setup guideline, there should not be any star imports?